### DISCUSSION TITLE
🔨iLogtail开发者双周会（2月22日 19:00-20:00）

### DISCUSSION THREAD

**Discussion by henryzhx8** [alibaba/ilogtail#659] [Announcements]
### 会议信息

时间：2月22日 周三 19:00 - 20:00
会议号：654 333 72626
电话呼入：+867388953959（中国大陆）+865965013656（中国大陆）
入会链接：https://meeting.dingtalk.com/j/PHHJvfEPgYw

### 议题

1. opentelemetry trace/metrics 协议兼容 @shunjiazhu 
2. 多仓库插件构建演示 @snakorse 
3. iLogtail自身可观测指标演进路线 @yyuuttaaoo 

### 会议纪要

1. 朱舜佳同学分享了Opentelemetry trace和metrics协议兼容相关的话题，详细介绍了如何改造现有的flusher_otlp_log插件以兼容iLogtail新版的metric协议等，同时以一个简单的配置示例介绍如何使用新版的flusher_otlp插件。
代码PR：https://github.com/alibaba/ilogtail/pull/646

2. 郭刚平同学分享了多仓库插件联合构建相关的内容，以详实的例子为大家演示如何将私有仓库中的插件同主代码库进行联合编译，从而构建私有版本的iLogtail。
整体设计：https://github.com/alibaba/ilogtail/discussions/618
代码PR：https://github.com/alibaba/ilogtail/pull/619
针对该议题，社区就“如何仅编译私有仓库中的部分插件”进行了简单交流，郭刚平同学也用实例进行了展示。

3. 余韬同学分享了iLogtail自身可观测指标演进路线的话题，详细介绍了iLogtail自身观测性的现状及不足，并就未来可观测性建设的整体蓝图以及短期的发展路线提出了自己的看法。此议题也引起了社区的广泛交流，特别的，大家就iLogtail自身可观测指标的暴露形式进行了深入的讨论，具体形式可待后期社区进一步交流定稿。

### 视频回放

https://www.bilibili.com/video/BV1Rb41197TD/?spm_id_from=333.999.0.0

**Comment by liuhaoyang** [comment]
我们分享两个Topic
1. 多仓库插件构建演示 @snakorse 
2. opentelemetry trace/metrics 协议兼容 @shunjiazhu
